ResearchKit/Common/ORKFormStep.m

```diff
@@ -84,6 +84,17 @@ - (void)validateParameters {
     for (ORKFormItem *item in _formItems) {
         [item.answerFormat validateParameters];
     }
+    
+    [self validateIdentifierUniquiness];
```

Comment:
Spelling: uniquiness -> uniqueness (everywhere)

Although actually I'd just call this method `validateIdentifiersUnique`